src/redux/category-new/category.sagas.js

```diff
@@ -0,0 +1,117 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
```

Comment:
Why do you name this folder category-new. Can you rename it?

Reply:
All ready renamed it